### ISSUE TITLE
Mega Thread: Tiny infernals

### ISSUE THREAD

**Issue by MatticusGib** [IllInuz/FelstormBT#37]
### Type bug

General

### Description

I found some tiny Infernals on interactable objects in the Legion Camp: Ruin in Aszuna during the Saving Stellagosa quest, but it occurs to me there are lots of random demons in places they aren't supposed to be, always just standing or floating, and not interactable.

### Expected behaviour

There should not be any random infernals or imps in random zones, next to in game objects we can interact with or otherwise.

### Steps to reproduce the problem

Basically, we need to track down everywhere these things are, and get them added to a mega thread for cleanup.

### Faction

Alliance

### Level, Class, and Spec

110 Human Brewmaster Monk

### Operating system

Windows 10 Pro x64

### Additional Notes

.
![WoWScrnShot_110823_114841](https://github.com/IllInuz/FelstormBT/assets/148009310/4985cde6-bd40-4c51-b03c-8ceb2d929c71)
